apps/src/feedback.js

```diff
@@ -883,13 +886,7 @@ FeedbackUtils.prototype.getGeneratedCodeElement_ = function(options) {
       options.generatedCodeDescription);
   var code = this.studioApp_.polishGeneratedCodeString(this.getGeneratedCodeString_());
 
-  var codeDiv = document.createElement('div');
-  codeDiv.innerHTML = require('./templates/code.html.ejs')({
-    message: infoMessage,
-    code: code
-  });
-
-  return codeDiv;
+  return (<Code message={infoMessage} code={code} />);
 };
```

Comment:
Ooooh, can we use the [JSX Spread operator](https://facebook.github.io/react/docs/jsx-spread.html) ?

``` javascript
var generatedCodeProperties = this.getGeneratedCodeProperties_(options);

return (<Code {...generatedCodeProperties} />);
```